src/chapters/control-flow-and-collections/hashmap.rst

```diff
@@ -205,7 +205,7 @@ References
 ----------
 
 -  `HashMap Class
-   (docs.oracle.com) <https://docs.oracle.com/en/java/javase/13/docs/api/java.base/java/util/HashMap.html>`__
+   (docs.oracle.com) <https://docs.oracle.com/en/java/javase/5/docs/api/java.base/java/util/HashMap.html>`__
```

Comment:
This link leads to a 404. Is the link supposed to be: https://docs.oracle.com/en/java/javase/15/docs/api/java.base/java/util/HashMap.html ?